Library/Formula/qbzr.rb

```diff
@@ -0,0 +1,25 @@
+class Qbzr < Formula
+  desc "Simple Qt cross-platform frontend for some of Bazaar commands"
+  homepage "https://launchpad.net/qbzr"
+  url "https://launchpad.net/qbzr/0.23/0.23.1/+download/qbzr-0.23.1.tar.gz"
+  sha256 "3211adef11c975dfbb6c80285651e2e6f3bfa99f1baa1a95371e8490ea8ff441"
+
```

Comment:
Mind adding `bottle :unneeded` to all these `bzr` plugins that _don't_ require any compile?

It should go one empty line beneath the `sha256` and one empty line above the `depends_on`.